plugin/httptrace/clienttrace.go

```diff
@@ -46,6 +49,9 @@ var (
 	HTTPHeaderMIME = tag.New("http.mime")
 	HTTPRemoteAddr = tag.New("http.remote")
 	HTTPLocalAddr  = tag.New("http.local")
+	messageKey     = tag.New("message",
```

Comment:
Should this also be exported as the other tags are?

Reply:
I have exported it for now but I wonder if other should be exported.